aspnetcore/signalr/get-started/sample/wwwroot/js/es5-chat.js

```diff
@@ -18,11 +13,11 @@ document.getElementById("sendButton").addEventListener("click", function (event)
     var user = document.getElementById("userInput").value;
     var message = document.getElementById("messageInput").value;
     connection.invoke("SendMessage", user, message).catch(function (err) {
-        return console.error;
+        return console.error(err.toString());
```

Comment:
nit: `err => console.error(err.toString())`

Reply:
@anurse  fix that and I'll get it merged to live.

Reply:
This is the ES5 transpilation, it doesn't use arrow functions. It's generated by Babel